OrbitCore/Utils.h

```diff
@@ -275,6 +275,25 @@ class CWindowsMessageToString {
 };
 #endif
 
+//-----------------------------------------------------------------------------
+inline std::string ShortenStringWithEllipsis(const std::string& text, int max_len) {
```

Comment:
I think the argument `text` can be of type `std::string_view`.

Reply:
Changed